src/components/FieldTextarea/index.tsx

```diff
@@ -1,77 +1,57 @@
-import { useEffect, useState } from 'react';
-
 import { Textarea, TextareaProps } from '@chakra-ui/react';
 import { FieldProps, useField } from '@formiz/core';
 
 import { FormGroup, FormGroupProps } from '@/components/FormGroup';
 
-export type FieldTextareaProps<FormattedValue = string> = FieldProps<
-  string,
+type Value = TextareaProps['value'];
+
+type UsualTextareaProps = 'placeholder';
+
+export type FieldTextareaProps<FormattedValue = Value> = FieldProps<
+  Value,
   FormattedValue
 > &
-  FormGroupProps & {
-    textareaProps?: Omit<
-      TextareaProps,
-      | 'id'
-      | 'value'
-      | 'name'
-      | 'defaultValue'
-      | 'onChange'
-      | 'onBlur'
-      | 'placeholder'
-    >;
-    autoFocus?: boolean;
+  FormGroupProps &
+  Pick<TextareaProps, UsualTextareaProps> & {
+    textAreaProps?: Omit<TextareaProps, UsualTextareaProps>;
```

Comment:
Not sure about the uppercase `A` as the name of the component is with a lowercase `a`.